server/activityItem/queries.ts

```diff
@@ -346,13 +349,17 @@ export const createPubReviewCommentAddedActivityItem = async (
 	reviewId: string,
 	threadCommentId: string,
 ) => {
-	const [review, threadComment]: [types.Review, types.ThreadComment] = await Promise.all([
+	const promises = await Promise.all([
 		ReviewNew.findOne({
 			where: { id: reviewId },
 		}),
 		ThreadComment.findOne({ where: { id: threadCommentId } }),
 	]);
-	const pub = await Pub.findOne({ where: { id: review.pubId } });
+
+	const review = expect(promises[0]);
+	const threadComment = expect(promises[1]);
```

Comment:
I do this un-destructuring in order to `expect` the result properly, otherwise I need to `expect` every instance where `review` is used. This way I only need to do it once.